src/install.sh

```diff
@@ -8,19 +8,19 @@ curl -sL https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL}/b
 -o /usr/local/bin/kubectl && chmod +x /usr/local/bin/kubectl
 kubectl version --client
 
-KUSTOMIZE=3.2.3
+KUSTOMIZE=3.3.1
 echo "downloading kustomize ${KUSTOMIZE}"
 curl -sL https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv${KUSTOMIZE}/kustomize_kustomize.v${KUSTOMIZE}_linux_amd64 \
```

Comment:
You need to change the download link, check https://github.com/kubernetes-sigs/kustomize/releases